README.md

```diff
@@ -31,6 +31,8 @@ deployment in a forked environment and interact with it to ensure it works as ex
 
 Start ganache.
 
+EDIT to eng: What's your.node.url.io? We need to spell this out in more detail. We should also have a link to Ganache and a one sentence explaination about what it is.
```

Comment:
We could change to `YOUR_NODE_URL`. Do you think it needs more detail?

Ganache is a pretty familiar tool to almost everyone doing ethereum development, but even if not, do you think they actually need to know what ganache is? I feel like it's somewhat irrelevant to the tutorial.

Thoughts?

Reply:
Changed to `YOUR_NODE_URL` and just added one sentence and a link to Infura to try to clarify things. Left this very brief, as I agree it doesn't need a lot of explanation. 

Also agreed Ganache is usually pretty common, but I don't see any harm in linking to it. Just added one sentence about Ganache with a link, so people can go read more elsewhere if curious.

Let me know what you think.